### PR TITLE
Bump Go versions and use '.x' to always get latest patch versions

### PR DESCRIPTION
**Pull request by HaraldNordgren**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,9 @@
 language: go
 
 go:
-  - go1.8
```
Comment (HaraldNordgren) [inline .travis.yml:4]: Removing 1.8 which is failing now with

```
$ go get -u github.com/golang/lint/golint
# golang.org/x/tools/go/internal/gcimporter
../../../golang.org/x/tools/go/internal/gcimporter/bexport.go:212: obj.IsAlias undefined (type *types.TypeName has no field or method IsAlias)
```

**Comment by HaraldNordgren** [comment]
@campoy Ping

File: .travis.yml (hunk 1 above)
Comment (campoy) [reply .travis.yml:4]: Probably worth dropping 1.9 too, I normally keep the latest 2 released versions + tip

**Review by campoy — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (HaraldNordgren) [reply .travis.yml:4]: @campoy Updated.

**Pull request closed without merging**